src/types/UIProtocol.ts

```diff
@@ -13,13 +13,23 @@ export enum ProtocolVersion {
   '0.0.1' = '0.0.1',
 }
 
-export enum ProtocolCommand {
+export enum MessageCode {
+  REQUEST = 0,
+  ANSWER = 1,
+  ERROR = 2,
+}
```

Comment:
That's not needed. The current UI protocol is very simple Request -> Response and that's all.